### ISSUE TITLE
After v4 image component is missing the alt text option when adding a image from the media library option.

### ISSUE THREAD

**Issue by jamescrabb** [soflyy/oxygen-bugs-and-features#2976]
**Any bug report that doesn't include a Sandbox link or replication steps will be closed. If you can't provide these pieces of information, we're happy to help you via our official support channel instead: https://oxygenbuilder.com/support. You need to select "Something's broken and I need help" from the dropdown box on the left.**

**Describe the bug**

It is no longer possible to add the alt text to images using the builder and in particular dynamic data.

This effects only the media library method of adding images to the image component. It would appear to be an oversight and would request it to be added back in.

**Steps required to reproduce the bug:**

1. Go to any page within the oxygen builder and add the basic image component
2. Click on "media library"
3. Scroll down through the options and you can see you can no longer add the alt text to the image.
4. Now select the "image url" option and you can see the desired result with the option to add dynamic data

Hope this is an oversight and can be reimplemented asap.

**Comment by andreslotta** [comment]
@jamescrabb This is expected behaviour. When you use the "Media Library" option the alt text is automatically fetched from the Media Library. If you need to override it in specific instances you can use the image element's attributes section. @Spellhammer recently talked about it in a video: https://www.youtube.com/watch?v=0zofUWbttmM

**Comment by Spellhammer** [comment]
@andreslotta Thanks for chiming in. You're absolutely correct and this is not a bug. Closing.
